docs/hiring-guide/community-spotlight/interview-celeste-horgan.rst

```diff
@@ -0,0 +1,85 @@
+Interview with Celeste Horgan
+=============================
+
+Celeste is a Canadian technical writer based in Berlin and has been writing documentation related to ecommerce for 4 years.
+
+Key takeaways
+*************
```

Comment:
We should keep these titles consistent across each set of posts. I prefer dashes (`---------`) as a 2nd level. Also we should keep a space under the headings to keep them consistent.